### PR TITLE
add readme for docker

### PR DESCRIPTION
**Pull request by jamescway**
thanks to @jfeltesse-mdsol!

### CODE REVIEW

File: doc/docker.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
+# Docker toolbox & docker-zipkin
```
Comment (jfeltesse-mdsol) [inline doc/docker.md]: you might want to add "on Mac OS X" because parts are mac os only (eg, `brew install` stuff)

**Comment by jfeltesse-mdsol** [comment]
:+1:

**Comment by codefromthecrypt** [comment]
Only suggestion I have is that it is probably in the wrong repo. Docs for
docker should be in docker-zipkin else we are surely to duplicate efforts.
Regardless, we can link out to this.

Make sense?

**Comment by jfeltesse-mdsol** [comment]
> Docs for docker should be in docker-zipkin

:+1:

**Pull request closed without merging**